ParameterDataApi.yaml

```diff
@@ -88,122 +144,79 @@ components:
           description: The success message provided back to the client
       required:
         - message
-        
-    ColumnDefinition:
-      type: array
-      items:
-        $ref: '#/components/schemas/ColumnElementDefinition'
-          
-    ColumnElementDefinition:
-      type: object
-      description: The column elements      
-      properties:
-        
-        name: 
-          description: The name of the column
-          type: string
-          pattern: '^[a-zA-Z0-9]{2,}'
-          
-        dataType:
-          $ref: '#/components/schemas/DataType'
-                   
-      required:
-        - name
-        - dataType
 
-    DataDefinition:
-      type: array
-      items: 
-        $ref: '#/components/schemas/DataElement'
-        
-    DataElement:
+    DataSample:
       type: object
+      description: A definition of a set of parameter data having a single discrete timestamp
       properties:
+      
         timestamp:
-          description: the timestamp of the sample
           type: string
           format: date-time
-            
-        fields:
-          type: array
-          items:
-            $ref: '#/components/schemas/ValueSpec'
-            
+          description: The timestamp that applies to the measurements in the data field (DataSample)
```

Comment:
What is the name of the structure in parenthesis at the end meant to convey?

Reply:
Will clarify in the description differently